lib/apiservers/engine/backends/container.go

```diff
@@ -985,22 +984,25 @@ func (c *Container) ContainerInspect(name string, size bool, version version.Ver
 	}
 	var started time.Time
 	var stopped time.Time
-	if results.Payload.ProcessConfig.StartTime != nil && *results.Payload.ProcessConfig.StartTime > 0 {
```

Comment:
This is not a cosmetic change. it a result of a variable becoming a value not a pointer during migration.